src/pages/index.js

```diff
@@ -22,35 +22,50 @@ import ServiceMeshFocussed from "../sections/Home/Service-mesh-focussed";
 import CloudNativeLeaders from "../sections/Home/Cloud-Native";
 import SoSpecial from "../sections/Home/So-Special-Section";
 
-const IndexPage = () => (
-  <ThemeProvider theme={theme}>
-    <GlobalStyle />
-    <SEO title="Layer5" description="Making service meshes available to the rest of us. Open source software for management of service meshes. Allowing developers to focus on business logic, not infrastructure concerns. Empowering operators to confidentally run modern infrastructure." />
-    <Layout>
-      <Navigation />
-      <Banner />
-      <Partners />
-      <Integrations />
-      {/* <Features /> */}
-      <ServiceMeshManagement />
-      <Statement />
-      <ProudMaintainers />
-      {/* <AppScreens /> */}
-      {/* <Pricing /> */}
-      {/* <Testimonial /> */}
-      <CloudNativeLeaders />
-      <StewardsOfIndustry />
-      {/* <Getapp /> */}
-      {/* <News /> */}
-      {/* <Faq category = "all"/> */}
-      {/*<ContactCard />*/}
-      <SoSpecial />
-      <ServiceMeshFocussed />
-      {/* <EngineerEnabler /> */}
-      <SubscribeSection />
-      <Footer />
-    </Layout>
-  </ThemeProvider>
-);
+const IndexPage = () => { 
+  const schema= {
+    "@context": "https://schema.org",
+    "@type": "Corporation",
+    "name": "Layer5",
+    "url": "https://layer5.io/",
+    "logo": "https://layer5.io/images/layer5-tag-dark-bg.png",
+    "sameAs": [
+      "https://twitter.com/layer5",
+      "https://www.linkedin.com/company/layer5/",
+      "https://github.com/layer5io"
```

Comment:
Perhaps, we might include https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA

Reply:
(for the YouTube channel)

Reply:
@Jashpatel1, I assume this will just be another entry in the array...

Reply:
Yes, @leecalcote it should be another entry. The same could be done for the Meshery website too. I have added Wikidata entries for both and once we have the website ready, we should get a knowledge panel in no time.

Reply:
That is fantastic. This is great.